building/tracks/new/add-first-exercise.md

```diff
@@ -25,3 +22,52 @@ The "Hello, World!" exercise has some special rules applied to it:
 - The stub file contains an almost-working implementation, but instead of "Hello, World!" it uses "Goodbye, Mars!"
 - It has no `prerequisites`
 - It has no `practices`
+
+### Updating the config
+
+Start by adding an entry to the `exercises.practice` array in the top-level `config.json` file.
+
+```
+{
+  "exercises": {
+    "practice": [
+      {
+        "uuid": "",
+        "slug": "hello-world",
+        "name": "Hello World",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
+      }
+    ]
+  }
+}
+```
+
+You can use the [Configlet][configlet] tool to get a UUID.
+Download Configlet by running `bin/fetch-configlet` from the root of the repository.
+Then generate a UUID using the `bin/configlet uuid` command.
+
+### Generating required files
+
+To implement the "Hello, World!" exercise, you'll need to create several files.
+Which files exactly is described in the [Practice Exercises documentation](/docs/building/tracks/practice-exercises).
+
+Most of the files can be added automatically by running Configlet's `sync` command:
+
+```
+bin/configlet sync --update --docs --metadata --tests
+```
+
+In addition to the generated files, you will to create a test suite, a stub solution that serves as the starting point for the student, and a sample solution that CI uses to ensure that the test suite is in a good state.
+
+In order to create these files you need to make some decisions about filenames and file paths.
+The rule of thumb is to use names that are idiomatic for the language, and where there are no strong preferences prefer shallower directory structures.
+The sample solution will need to be identifiable by the CI script, so it's advisable to choose a generic basename that all exercises can use, e.g. `example`, `sample`, or `reference-solution`.
+
```

Comment:
I wonder if we should also steer people towards populating the root `config.json`'s `files` property, which can be used as a template for the syncing of the filepaths later on. But maybe that should be a separate item and be part of the next couple of exercises (makes more sense there probably). We could then also point people towards using the same `configlet sync` commands, but those would then also populate the files.

Reply:
I'm really torn. Getting configlet to sync those files is really handy, but I think coming up with the file names in the context of implementing a concrete exercise is a bit better than coming up with the paths up front and getting configlet to populate the files.

> But maybe that should be a separate item and be part of the next couple of exercises (makes more sense there probably).

I think that's my conclusion as well.